staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -68,8 +68,8 @@ type Config struct {
 	// An underlying storage.Versioner.
 	Versioner storage.Versioner
 
-	// The Cache will be caching objects of a given Type and assumes that they
-	// are all stored under ResourcePrefix directory in the underlying database.
+	// ResourcePrefix directory stores the empty object, which stores an object of type Type, 
+	// created by NewFunc function in the underlying database.
```

Comment:
If I understand correctly, the concept of `Type` was removed by an earlier PR, and that's what the PR is to clean up?

Reply:
Yes, the type was replaced by NewFunc which created the empty objects rather than initialising objects of type Type.

Reply:
But the object type stayed the same as far as I understood. Am I missing something?

Reply:
Right. The issue is that there is no visible concept of a `Type` anymore. So the documentation comment should also be agnostic of it imo

Reply:
So, should its mention also be removed from here ?
https://github.com/kubernetes/apiserver/blob/89d2b256c349757c5ec29b3f33420caa41efcb5d/pkg/storage/cacher/cacher.go#L89

Reply:
I think so, yes.

Reply:
Made the change, please review it.